### DISCUSSION TITLE
The Plotnine Adventure: A Journey Through Gene Expression 🧬🎨

### DISCUSSION THREAD

**Discussion by PablaOO7** [has2k1/plotnine#858] [2024 Plotnine Contest]
### Authors

Jaspreet Singh Pabla

### Full description

README Link: https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/Third_Entry_Plotnine/README.md

### Plots: All plots are available in the repo in the png format.

Plots Link: https://github.com/PablaOO7/Plotnine-Contest-2024/tree/main/Third_Entry_Plotnine/Plots

### Code repository

Code Link: https://github.com/PablaOO7/Plotnine-Contest-2024/blob/main/Third_Entry_Plotnine/p9_biological_data_analysis_notebook.ipynb

**Comment by has2k1** [comment]
![pca_facet_plot](https://github.com/user-attachments/assets/76440e13-01d3-49bd-8235-77f6c84adcae)
<hr/>

![volcano_plot](https://github.com/user-attachments/assets/312d92e2-61f8-4a47-b33f-235f61908b9a)
<hr/>

![ma_plot](https://github.com/user-attachments/assets/a5b0f7e3-b76f-45fc-a7f7-307e108fca02)
<hr/>

![facet_heatmap](https://github.com/user-attachments/assets/c916bb6e-5d9a-41fd-8c18-5fc8ebc01f29)

**Comment by EconomiCurtis** [comment]
Ahoy @PablaOO7 , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Reply by PablaOO7** [reply]
Hi @EconomiCurtis @has2k1,
It has been a while since the contest, and I was wondering if there is an update on when we can expect the results.

**Reply by has2k1** [reply]
The results will be announced within 2 days.

**Comment by has2k1** [comment]
@PablaOO7, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you again for taking part in the contest.
